### PR TITLE
Update URBANopt Project

### LINKED ISSUE #361: CLI command to update an existing project to latest URBANopt version

### PR DESCRIPTION
**Pull request by tanushree04**
### Pull Request Description

- Adds update command to replace files in project directory with latest files in the example_files folder.
- To use, `bundle exec uo update --existing-project-folder (name of project folder) --new-project-directory (location of project directory)`

### Checklist (Delete lines that don't apply)

- [x] Unit tests have been added or updated
- [ ] All ci tests pass (green)
- [x] An [issue](https://github.com/urbanopt/urbanopt-cli/issues) has been created (which will be used for the changelog)
- [x] This branch is up-to-date with develop

### CODE REVIEW

**Comment by tanushree04** [comment]
Looking into why the OpenDSS test is failing.

**Comment by tanushree04** [comment]
@kflemin this should be ready for review

File: lib/uo_cli.rb [hunk 1]
```diff
@@ -677,6 +693,57 @@ def self.create_project_folder(dir_name, empty_folder = false, overwrite_project
...
+          if Dir.exist?(File.join(path, 'opendss'))
+            Pathname.new(File.join(path_item, 'opendss')).children.each { |opendss_file| FileUtils.cp_r(opendss_file, File.join(new_path, 'opendss'), remove_destination: true) }
+          end
+
+          # Replace DISCO files
+          if Dir.exist?(File.join(path, 'disco'))
+            Pathname.new(File.join(path_item, 'disco')).children.each { |disco_file| FileUtils.cp_r(disco_file, File.join(new_path, 'disco'), remove_destination: true) }
+          end
+
+          # TODO: Replace geojson files?
```
Comment (tanushree04) [inline lib/uo_cli.rb:742]: @kflemin any thoughts on how we should copy the feature file? These would vary by the project we create, should we determine the project time and then copy the relevant one?

Comment (kflemin) [reply lib/uo_cli.rb:742]: yes I think that would work: check that each "example" project json file exists, if it does, replace with the newer copy?

**Review by kflemin — APPROVED** [review]

**Pull request merged**